client/src/tests/recipe.test.js

```diff
@@ -1,97 +1,98 @@
 import { render } from '@testing-library/react';
 import { search } from 'components/searchBar';
```

Comment:
Certainly not worth rejecting the PR over, but it's interesting to note that some things are camelcase and some are snake.